test/test.js

```diff
@@ -177,7 +168,7 @@ describe("live demo", () => {
 
     const issuesMessage = await (await getWaiting(By.css(".issue-count-banner"))).getText();
 
-    expect(issuesMessage).toMatch(/[\d]\s*words and [\d]\s*issues/); //expect 4 but let's be more tolerant and allow 3 and 5 as well ;-)
+    expect(issuesMessage).toMatch(/[\d]\s*words and [\d]\s*issue[s]?/); //expect 4 but let's be more tolerant and allow 3 and 5 as well ;-)
```

Comment:
two comments regarding this regex:
1) it will also match "issue" (singular), however the comment suggests we would like to accept 3-5 but not 1?
2) ? makes the preceding element zero or one times. But there should be no reason to create a character class "[s]?" (since there is only a single character in this character class, not a range of characters or so), just "s?" should be enough.

Reply:
1. I removed the outdated comment.
2. Simplified regex to use `issues?`